app/context/physical_quantity.py

```diff
@@ -0,0 +1,528 @@
+                    res = parameters["reserved_expressions"]["response"]["standard"]["value"]
+                if (ans is not None and ans.is_constant()) and (res is not None and res.is_constant()):
+                    if parsing_params.get('rtol', 0) > 0:
+                        value_match = bool(abs(float((ans-res)/ans)) < parsing_params['rtol'])
+                    elif parsing_params.get('atol', 0) > 0:
+                        value_match = bool(abs(float(ans-res)) < parsing_params['atol'])
+
+        substitutions = [(key, expr["standard"]["unit"]) for (key, expr) in reserved_expressions]
+        unit_match = is_equal(lhs, rhs, substitutions)
+
+        output_tags = None
+        if value_match is True and unit_match is True:
+            output_tags = {
+                label+"_TRUE": {"lhs": lhs_string, "rhs": rhs_string}
+            }
+        else:
+            output_tags = {
+                label+"_FALSE": {"lhs": lhs_string, "rhs": rhs_string}
+            }
+
+        return output_tags
+
+    def dimension_match(unused_inputs):
+        substitutions = [(key, expr["dimension"]) for (key, expr) in reserved_expressions]
+        dimension_match, _ = is_proportional(lhs, rhs, substitutions)
+
+        output_tags = None
+        if dimension_match is True:
+            output_tags = {
+                label+"_DIMENSION_MATCH"+"_TRUE": {"lhs": lhs_string, "rhs": rhs_string}
+            }
+        else:
+            output_tags = {
+                label+"_DIMENSION_MATCH"+"_FALSE": {"lhs": lhs_string, "rhs": rhs_string}
+            }
+        return output_tags
+
+    def unit_comparison(unused_inputs):
+        substitutions = [(key, expr["original"]["unit"]) for (key, expr) in reserved_expressions]
+
+        if is_equal(lhs, rhs, substitutions) is True:
+            output_tags = {
+                label+"_UNIT_COMPARISON"+"_IDENTICAL": None
+            }
+        else:
+            local_substitutions = [(key, expr["quantity"].expanded_unit) for (key, expr) in reserved_expressions]
+            result, ratio = is_proportional(lhs, rhs, local_substitutions)
+            if result is True and ratio >= 1000:
+                output_tags = {
+                    label+"_UNIT_COMPARISON"+"_PREFIX_IS_LARGE": None
+                }
+            elif result is True and ratio <= 1/1000:
+                output_tags = {
+                    label+"_UNIT_COMPARISON"+"_PREFIX_IS_SMALL": None
+                }
+            else:
+                output_tags = {
+                    label+"_UNIT_COMPARISON"+"_SIMILAR": None
+                }
+
+        return output_tags
+
+    graph.add_evaluation_node(
+        label,
+        summary=f"Do the quantities {inputs[0]} and {inputs[1]} match?",
+        details=f"Converts {inputs[0]} and {inputs[1]} match to a common set of base units and compares their values.",
+        evaluate=quantity_match
+    )
+    graph.attach(
+        label,
+        label+"_TRUE",
```

Comment:
I prefer the labels to be set explicitly for now, since I am not sure what is the best label format, and if there are different ways that are more suitable for different situations.